app/scripts/modules/core/src/pipeline/config/triggers/pubsub/pubsubTrigger.html

```diff
@@ -25,13 +25,25 @@
     </div>
   </div>
 
+  <hr>
+
   <div class="form-group">
-    <div class="col-md-3 sm-label-right">
-      Constraints
+    <div class="col-md-4 sm-label-right">
+      Payload Constraints
       <help-field key="pipeline.config.trigger.pubsub.constraints"></help-field>
     </div>
     <div class="col-md-6">
-      <map-editor model="vm.trigger.constraints" add-button-label="Add constraint"></map-editor>
+      <map-editor model="vm.trigger.constraints" add-button-label="Add payload constraint"></map-editor>
```

Comment:
Should these be `payloadConstraints` now?

Reply:
Just to make clear what's what.

Reply:
Change in webhooks as well?

Reply:
Good call

Reply:
Idk actually, this will break people using pipeline templates with webhook and pubsub triggers.